src/redux/books/books.js

```diff
@@ -1,25 +1,112 @@
 const ADD_BOOK = 'bookStore/books/ADD_BOOK';
 const REMOVE_BOOK = 'bookStore/books/REMOVE_BOOK';
+const STARTED_BOOK = 'bookStore/books/STARTED_BOOK';
+const MANAGE_BOOK_FAILURE = 'bookStore/books/ADD_BOOK_FAILURE';
 
-export const addBook = (payload) => ({
+const appIdentifier = 'oFzStVgFLC3mbAjB7OrJ';
+const urlApi = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi';
+
+const addBook = (payload) => ({
   type: ADD_BOOK,
   payload,
 });
 
-export const removeBook = (payload) => ({
+const removeBook = (payload) => ({
   type: REMOVE_BOOK,
   payload,
 });
 
-const booksReducer = (state = [], action) => {
+const manageBookStarted = () => ({
+  type: STARTED_BOOK,
+});
+
+const manageBookFailed = (error) => ({
+  type: MANAGE_BOOK_FAILURE,
+  payload: {
+    error,
+  },
+});
+
+const addBookToApi = (data) => {
```

Comment:
- [ ] Please make sure  `add book` and `remove book` features should persist all changes in the server, refreshing the page after adding books, the books no longer displays, please kindly fix that.